Homework17/Tests/UsersTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Homework17.Helpers;
+using Homework17.Models.JsonModels;
+using Homework17.Services;
+using NUnit.Framework;
+using System.Net;
```

Comment:
Please move System usings to the first place

Reply:
oh yeeeees

Reply:
i removed it at all